laghos_rom.cpp

```diff
@@ -353,9 +353,22 @@ ROM_Basis::ROM_Basis(ROM_Options const& input, Vector const& S, MPI_Comm comm_,
         initV = new CAROM::Vector(tH1size, true);
         initE = new CAROM::Vector(tL2size, true);
 
-        //TODO: Tony param init after PR 77
-        if (input.paramOffset)
+        // std::string path_init = (parameterID >= 0) ? basename + "/ROMoffset/param" + std::to_string(parameterID) + "_init" : basename + "/ROMoffset/init"; // TODO: Tony PR77
+        std::string path_init = basename + "/ROMoffset/init";
+
+        if (input.restore || (input.offsetType == saveLoadOffset && !input.paramOffset))
+        {
+            // Read offsets in the restore phase or in the online phase of non-parametric save-and-load mode
+            initX->read(path_init + "X" + std::to_string(input.window));
+            initV->read(path_init + "V" + std::to_string(input.window));
+            initE->read(path_init + "E" + std::to_string(input.window));
+
+            cout << "Read init vectors X, V, E with norms " << initX->norm() << ", " << initV->norm() << ", " << initE->norm() << endl;
+        }
+        else if (input.offsetType == saveLoadOffset && input.paramOffset)
         {
+            // TODO: Tony interpolation PR 77
```

Comment:
Should this TODO still be here?

Reply:
Please ignore those TODOs. They are to be done in another PR and just left there to remind me.